.github/stale.yml

```diff
@@ -0,0 +1,4 @@
+daysUntilStale: 60
+daysUntilClose: 7
+onlyLabels:
```

Comment:
what does onlyLabels do? can we add a comment?

Reply:
At the beginning, we are only checking the issues that containing this label.